### ISSUE TITLE
(Feature Request) Pass ref object as key directly

### ISSUE THREAD

**Issue by luminisward** [Kong/swrv#233]
```
const params = ref('1')
const { data } = useSWRV(params, fetcher)
```
It works fine, just has a type error.

```
export declare type IKey = keyFunction | keyType | WatchSource<keyType>;
```
Append `WatchSource` to `IKey` to resolve.

**Comment by darrenjennings** [comment]
sounds like it's just a typing issue? Do you feel up to submitting a PR?

**Comment by luminisward** [comment]
> sounds like it's just a typing issue? Do you feel up to submitting a PR?

No problem. I have created a PR.

**Comment by darrenjennings** [comment]
closed by #238 thanks @luminisward !
